docs/_user_guide/best_practices_jupyter_does_not_work.md

```diff
@@ -28,7 +41,7 @@ You can delete your `~/.jupyter/` directory, and when you restart JupyterHub, an
 If you cannot write a file to the directory you're in, then JupyterHub will have trouble launching notebooks or consoles.  If you're unsure, navigate to your home directory and try again!
 
 ## Have you installed any libraries recently which may conflict with JupyterHub? 
-This can present as a 404 error. Launch JupyterHub from the terminal with `/opt/jupyter/bin/jupyter-lab` to check this. If everything is running correctly, you should get an indication of a successful boot in your terminal. If there are errors reguarding specific libraries, you may want to uninstall those libraries.
+This can present as a 404 error.  ssh into the terminal and Launch JupyterHub with `/opt/jupyterhub/bin/jupyter-lab` to check this. If everything is running correctly, you should get an indication of a successful boot in your terminal. If there are errors reguarding specific libraries, you may want to uninstall those libraries.
```

Comment:
Format looks weird. "This can present as a 404 error. ssh into the terminal and Launch JupyterHub"